homeassistant/components/mullvad/__init__.py

```diff
@@ -0,0 +1,68 @@
+"""The Mullvad VPN integration."""
+import asyncio
+from datetime import timedelta
+import logging
+
+import async_timeout
+from mullvad_api import MullvadAPI
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import update_coordinator
+
+from .const import DOMAIN
+
+PLATFORMS = ["binary_sensor", "sensor"]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the Mullvad VPN integration."""
+    # Make sure coordinator is initialized.
+    await get_coordinator(hass)
```

Comment:
This can be done in the async_setup_entry. There is no real use to the async_setup in this case.

Reply:
I can move the get_coordinator to `async_setup_entry`, but i think i still need `async_setup` otherwise hass complains that there is `no setup function defined`.